packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -52,8 +58,18 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
         return mergeSchemas({ schemas: [modelsSchema, schema] });
     }
 
+    public generateResources(metadata: GraphbackCoreMetadata): void {
+        const schemString = this.transformSchemaToString(metadata.getSchema());
+        if (!existsSync(this.pluginConfig.outputPath)) {
+            mkdirSync(this.pluginConfig.outputPath, { recursive: true });
+        }
+        const location = resolve(this.pluginConfig.outputPath, `schema.${this.pluginConfig.format}`);
```

Comment:
```suggestion
        const location = resolve(this.pluginConfig.outputPath, `generated.${this.pluginConfig.format}`);
```

Should be `generated.ts`

Reply:
Hmm.. Actually previously we were hardcoding folder name to ./schema/generated.ts since now we do not enforce anything like it we can have just `schema.ts` in whatever location we want